synapse/lib/storm.py

```diff
@@ -950,12 +949,12 @@ def _stormOperJoinTags(self, query, oper):
         tags = {tag for node in nodes for tag in s_tufo.tags(node, leaf=True)}
 
         if not forms:
-            forms = core.getModelDict().get('forms')
+            forms = set(core.getModelDict().get('forms'))
```

Comment:
This is kind of equivalent to the getTufoForms() API from the datamodel which gets the currently registered forms; however a more complete set of data is to life all the syn:form notes and get data from them (in the event that there are forms in the cortex which don't have their models registered with the current datamodel in memory).